var/spack/repos/builtin/packages/parsec/package.py

```diff
@@ -20,6 +20,7 @@ class Parsec(CMakePackage, CudaPackage):
     maintainers = ['abouteiller', 'bosilca', 'herault']
 
     version('master', branch='master')
+    version('3.0.2012', sha256='f565bcfffe106be8237b6aea3e83a5770607b7236606414b6f270244fa6ec3bc', preferred=True)
```

Comment:
Can we remove the release candidate too? Or is there a compelling reason to keep it?